x-pack/plugins/security_solution/public/detections/components/host_isolation/use_host_isolation_action.test.tsx

```diff
@@ -116,8 +129,8 @@ describe('useHostIsolationAction', () => {
     it(`${name} is invoked as 'enabled' when Crowdstrike alert and FF enabled`, () => {
       render('crowdstrike');
 
-      expect(hook).toHaveBeenCalledWith([''], 'sentinel_one', {
-        enabled: false,
+      expect(hook).toHaveBeenCalledWith(['expectedCrowdstrikeAgentId'], 'crowdstrike', {
```

Comment:
These changes do not drop SentinelOne tests. Just the values were temporarily defaulted to SentinelOne before (introduced in Action Client PR). Now correctly expecting crowdstrike agentType, and `expectedCrowdstrikeAgentId` is just a value for agent.id

Reply:
got it. thank you... So where are the checks that sentinelone one is also working as expected? Because your changes seem to be checking for crowdstrike only

Reply:
The same file, there are 2 crowdstrike, and 2 sentinel_one tests.